gateway/integration_tests/src/main.rs

```diff
@@ -571,6 +571,11 @@ async fn liquidity_test() -> anyhow::Result<()> {
             gw_send.pay_invoice(invoice).await?;
         }
 
+        info!("Testing paying through LND Gateway...");
+        let invoice = gw_ldk.create_invoice(1_550_000).await?;
```

Comment:
How are the gateways connected? Something like ldk <-> lnd <-> cln ?

Reply:
Yes, exactly.

Eventually cln will be replaced with another ldk.